alicloud/extension_cms.go

```diff
@@ -1,9 +1,10 @@
 package alicloud
 
 const (
-	Average = "Average"
-	Minimum = "Minimum"
-	Maximum = "Maximum"
+	Average          = "Average"
+	Minimum          = "Minimum"
+	Maximum          = "Maximum"
+	ErrorCodeMaximum = "ErrorCodeMaximum"
```

Comment:
There is not `ErrorCodeMaximum` in the official docs: https://www.alibabacloud.com/help/doc-detail/28619.htm. Please check it.

Reply:
This is true but this is due to the fact that `acs_networkmonitor ` is not documented at all. It is completely missing from the docs.